src/Hometask_2/Task3.java

```diff
@@ -0,0 +1,18 @@
+package Hometask_2;
+
+public class Task3 {
+    public static void main(String[] args) {
+        task3();
+    }
+    public static void task3(){
+        for(int i = 0; i < 4; i++){
+            if (i < 3){
+                System.out.println("*****");
+                System.out.println();
+            }
+            else {
+                System.out.print("*****");
+            }
+        }
```

Comment:
public static void main(String[] args) {
        for(int i = 0; i < 4; i++){
            for (int j = 0; j < 5; j++){
                System.out.print("*");
            }
            System.out.println();
        }
    }